src/common/seed/images.ts

```diff
@@ -81,18 +81,26 @@ const generateRandomImage = () => {
 	return selectedVariant;
 };
 
-const generateBuild = () => {
+const generateBuild = (forStatic?: boolean, counter?: number) => {
 	const statuses = ['building', 'error', 'ready'];
 	const randomNum = Math.round((Math.random() * 2));
 
+	if (forStatic && counter !== undefined) {
```

Comment:
What does this counter field actually mean? Change it to be statusIndex instead of counter